### ISSUE TITLE
RFID not detecting some card types that other devices detect (e.g. Keysy) 

### ISSUE THREAD

**Issue by PsiPhiTheta** [flipperdevices/flipperzero-firmware#3870]
### Describe the bug.

F0 is refusing to read some RFID cards (but working on others). I see a lot of posts about RFID functionality on the flipper being rather temperamental. I used to use the Keysy RFID cloner which works absolutely fine for everything I throw at it but I see some protocols seem to not have been programmed on flipper? Is that right? I will attach the raw reads to this issue for your reference. I also tried many times, completely still on a table, free from interference etc. 

### Reproduction

1. Switch on
2. Press RFID
3. Reading ASK / PSK scans continually without detecting anything 
[413psk.txt](https://github.com/user-attachments/files/16848016/413psk.txt)
[413ask.txt](https://github.com/user-attachments/files/16848017/413ask.txt)

### Target

Keysy rewritable key fob

**Comment by bettse** [comment]
> F0 is refusing to read some RFID cards (but working on others)

Yes, there are 125khz RFID that the flipper doesn't support.  Some for hardware reasons, others because they haven't been implemented yet.

> I see some protocols seem to not have been programmed on flipper? Is that right?

Correct.  The keysy works differently than the flipper zero, and the list of supported protocols is different.

> I will attach the raw reads

It would be most helpful to know the brand/protocol/format being used.  Possibly the raw data isn't necessary, and it is just a format or protocol that hasn't been implemented due to a lack of popularity.
